[Doc] [S3 File] doc says the`schema` field is optional, but is necessary in the code

### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

The doc(https://seatunnel.apache.org/docs/2.3.9/connector-v2/source/S3File/#text-or-csv-file-type) says the  schema is optional, but actually is not:
Caused by: org.apache.seatunnel.api.configuration.util.OptionValidationException: ErrorCode:[API-02], ErrorDescription:[Option item validate failed] - There are unconfigured options, the options('schema') are required because ['file_format_type' == TEXT || 'file_format_type' == JSON || 'file_format_type' == EXCEL || 'file_format_type' == CSV || 'file_format_type' == XML] is true.
	at org.apache.seatunnel.api.configuration.util.ConfigValidator.validate(ConfigValidator.java:200)


### SeaTunnel Version

2.3.9

### SeaTunnel Config

```conf
none
```

### Running Command

```shell
none
```

### Error Exception

```log
Caused by: org.apache.seatunnel.api.configuration.util.OptionValidationException: ErrorCode:[API-02], ErrorDescription:[Option item validate failed] - There are unconfigured options, the options('schema') are required because ['file_format_type' == TEXT || 'file_format_type' == JSON || 'file_format_type' == EXCEL || 'file_format_type' == CSV || 'file_format_type' == XML] is true.
	at org.apache.seatunnel.api.configuration.util.ConfigValidator.validate(ConfigValidator.java:200)
```

### Zeta or Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/docs/en/connector-v2/source/OssFile.md
+++ b/docs/en/connector-v2/source/OssFile.md
@@ -91,25 +91,11 @@ connector will generate data as the following:
 
 ### Text Or CSV File Type
 
-If you assign file type to `text` `csv`, you can choose to specify the schema information or not.
+If you set the `file_format_type` to `text`,`excel`,`csv`,`xml`. Then it's required to set the `schema` field to tell connector how to parse data to the row.
 
-For example, upstream data is the following:
+If you set the `schema` field, you should also set the option `field_delimiter`, except the `file_format_type` is `csv`, `xml`, `excel`
 
-```text
-
-tyrantlucifer#26#male
-
-```
-
-If you do not assign data schema connector will treat the upstream data as the following:
-
-|        content        |
-|-----------------------|
-| tyrantlucifer#26#male |
-
-If you assign data schema, you should also assign the option `field_delimiter` too except CSV file type
-
-you should assign schema and delimiter as the following:
+you can set schema and delimiter as the following:
 
 ```hocon
 

--- a/docs/en/connector-v2/source/S3File.md
+++ b/docs/en/connector-v2/source/S3File.md
@@ -97,25 +97,11 @@ connector will generate data as the following:
 
 ### Text Or CSV File Type
 
-If you assign file type to `text` `csv`, you can choose to specify the schema information or not.
+If you set the `file_format_type` to `text`,`excel`,`csv`,`xml`. Then it's required to set the `schema` field to tell connector how to parse data to the row.
 
-For example, upstream data is the following:
+If you set the `schema` field, you should also set the option `field_delimiter`, except the `file_format_type` is `csv`, `xml`, `excel`
 
-```text
-
-tyrantlucifer#26#male
-
-```
-
-If you do not assign data schema connector will treat the upstream data as the following:
-
-|        content        |
-|-----------------------|
-| tyrantlucifer#26#male |
-
-If you assign data schema, you should also assign the option `field_delimiter` too except CSV file type
-
-you should assign schema and delimiter as the following:
+you can set schema and delimiter as the following:
 
 ```hocon
 

